pkg/common/unittestutils/unittestutils.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Copyright 2020. since it is a new file.

I would suggest renaming as `pkg/common/unittestscommon/utils.go` or something better.